<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddableListWidget</class>
 <widget class="QWidget" name="AddableListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListView" name="listView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_Add">
          <property name="text">
           <string>追加(&amp;%1)...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Del">
          <property name="text">
           <string>削除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Description">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Add</sender>
   <signal>clicked()</signal>
   <receiver>AddableListWidget</receiver>
   <slot>onClick_pushButton_Add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Del</sender>
   <signal>clicked()</signal>
   <receiver>AddableListWidget</receiver>
   <slot>onClick_pushButton_Del()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>AddableListWidget</receiver>
   <slot>onClick_listView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClick_pushButton_Add()</slot>
  <slot>onClick_pushButton_Del()</slot>
  <slot>onClick_listView()</slot>
 </slots>
</ui>
